deploy/kubernetes/resources/chirper/front-end-statefulset.json

```diff
@@ -24,16 +24,16 @@
             ],
             "env": [
               {
-                "name": "CASSANDRA_SERVICE_NAME",
-                "value": "_native._tcp.cassandra.default.svc.cluster.local"
-              },
-              {
                 "name": "WEB_BIND_PORT",
                 "value": "9000"
               },
               {
                 "name": "WEB_BIND_IP",
                 "value": "0.0.0.0"
+              },
+              {
+                "name": "APPLICATION_SECRET",
+                "value": "web-application-secret"
```

Comment:
I'll reiterate my concern that by hardcoding easily-guessable secrets in the example deployment configuration, we might encourage users to do the same, which is not safe for a production service. I'd rather put something like "changeme" in there. At least it will make it more clear that it shouldn't be used verbatim.